### ISSUE TITLE
Conan support

### ISSUE THREAD

**Issue by mjvankampen** [FreeYourSoul/FSeam#19]
**Is your feature request related to a problem? Please describe.**
CPP packages are always somewhat hard to manage and integrate.

**Describe the solution you'd like**
It would be nice to get your ok and support to create an FSeam package for conan, a CPP package manager.

**Describe alternatives you've considered**
Other package managers can be used as well, but they are less widely used, platform specific or just not very nice.

**Comment by FreeYourSoul** [comment]
You have my ok at 100% to integrate FSeam with Conan. It is something I planned to do if I got more time at hand.
But as FSeam currently works only on Linux, I thought Conan may reject it (not cross-platform)

**Comment by mjvankampen** [comment]
Cool, so it works on windows with mingw and VS. I only have an issue that your use of the install folder share is not very common. It is not allowed by the [conan-center-index](https://github.com/conan-io/conan-center-index/wiki/Error-Knowledge-Base#KB-H013). Before I start doing tricks in the conan recipe I would like to see why it is different for FSeam than for the other packages in CCI.

**Comment by FreeYourSoul** [comment]
I cleaned up and made the upgrade of CMake findPython3 works on travis. Do you want help on your side?

**Comment by mjvankampen** [comment]
Ah great! The question above still stands. Is putting everything in the share folder necessary? It is different from most other packages such as Catch for example.

**Comment by FreeYourSoul** [comment]
I don't think it is, it was to install correctly FSeam on my distro, but kind of a hotfix as I didn't think it had to standardize it ^^

**Comment by mjvankampen** [comment]
Ok I will try to fix that soon (life got in the way :-))

**Comment by daravi** [comment]
@mjvankampen I was wondering if you have had any chance to work on this?

This is what I have created so far:
https://github.com/daravi/conan-fseam
